### ISSUE TITLE
[Request]Need a way to implement my Unity video in browser with npm deploy.

### ISSUE THREAD

**Issue by praveenjha-github** [Unity-Technologies/UnityRenderStreaming#613]
Need a way to implement my Unity video in browser with npm deploy.

When, i run the WebBrowserInput  in unity
![webbrowser](https://user-images.githubusercontent.com/86768166/146320067-f87d7776-d95e-41bb-830f-7da59a97d36f.jpg)

and downloading the webage from unity itself, so i could find the Streaming Video in my Browser. with the cmd runnig as 'webserver.exe -w'
![unityWebPage](https://user-images.githubusercontent.com/86768166/146320193-c7ea4b43-a453-4c7e-a945-209f0362e5e7.jpg)

For implementing this in chrome browser, i have also deploy Node.js and npm. Also the Signaling Protocol is changed in 'UnityRenderingStreaming Project' from "http" to "websocket" 
![websocket](https://user-images.githubusercontent.com/86768166/146332297-ddbe5628-35aa-47f4-b841-5a5155cf83b4.jpg)

but could not get my Video stream in the chrome browser from unity, its just showing the blank screen.
![noStreamingVideo](https://user-images.githubusercontent.com/86768166/146319622-2493c2df-5474-47ee-99be-ba788b160b16.jpg)

the cmd is showing perfect with the npm
![cmd with npm](https://user-images.githubusercontent.com/86768166/146319912-3f408f4e-d608-4bc3-9a75-ec6408ce0bae.jpg)

i have also checked the Unity Signaling Protocol, its stating WebSocket
![unity webSocket](https://user-images.githubusercontent.com/86768166/146319722-6fff0f90-8d93-46ab-8fc3-3f16f59b7d62.jpg)

So, basically when i build the "UnityRenderStreaming" project i am not getting my Streaming video. I have also not made changes to the project.
Could you please help me with this?

**Comment by kannan-xiao4** [comment]
@praveenjha-github 
I think your package version is 3.0.1, but the latest is 3.1.0-exp.2, so please try again on latest version.

If it still happens, please tell us more about your environment. ( OS, browser, Unity version etc.)

**Comment by kannan-xiao4** [comment]
@praveenjha-github 
I will close this issue if there is no reply until next week. Thanks.
